pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -111,8 +117,14 @@ stdenv.mkDerivation rec {
       sha256 = "sha256-oC+bRjEHixv1QEFO9XAm4HHOwoiT+NkhknKGPydnZ5E=";
       revert = true;
     })
+    # Adds binfmt P support, activated when argv[0] ends with `-binfmt-P`.
+    (fetchpatch {
+      url = "https://salsa.debian.org/qemu-team/qemu/-/raw/master/debian/patches/linux-user-binfmt-P.diff";
+      sha256 = "sha256-oWMF0hbpzskus/1Ga4n1n868qx7PbY6nNjVHLZoOMeA=";
+    })
   ]
-  ++ lib.optional nixosTestRunner ./force-uid0-on-9p.patch;
+  ++ lib.optional nixosTestRunner ./force-uid0-on-9p.patch
+  ++ lib.optional stdenv.hostPlatform.isStatic ./aio-find-static-library.patch;
```

Comment:
What would it take for this to be upstreamable?

Reply:
This seems to be [already gone](https://github.com/qemu/qemu/blob/ac84b57b4d74606f7f83667a0606deef32b2049d/meson.build#L856) since https://github.com/qemu/qemu/commit/063d5119c18af76b9e3303856dd114b36a8e1a43 and should land in QEMU 8.1. I'll add a comment about this.